packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-set.spec.ts

```diff
@@ -426,8 +426,8 @@ moduleIntegrationTestRunner<IPricingModuleService>({
             priceSetBefore.id,
             {
               prices: [
-                { amount: 100, currency_code: "USD" },
-                { amount: 200, currency_code: "EUR" },
+                { amount: 100, currency_code:"usd" },
```

Comment:
info: you still have a lot of formatting issues in the pr's, we might need to look at it, the risk is that everytime for example I go into the same file I ll end up with hundreds of unexpected changes unrelated to my work because my editor will format everything 😄